### ISSUE TITLE
   "exit_profit_only": true is not working? 

### ISSUE THREAD

**Issue by enn36** [freqtrade/freqtrade#8204]
Hi Dev,

Doesn't config over-ride strategy? I had a stop-loss despite having "exit_profit_only": true in config.json

Using Version: docker-2023.2.dev-62e120a6

**Comment by stash86** [comment]
exit_profit_only don't override stoploss.

**Comment by enn36** [comment]
I see, thanks. How about stoploss = -1.00 then?

**Comment by enn36** [comment]
Searched discord and got an answer, sort of. 

Will test it out with -1.00 or  -0.99. Thanks
